UI/CharactersArtworkPanel/CharactersArtworkPanel.gd

```diff
@@ -108,32 +111,34 @@ func updateImages():
 		textureRect3.texture = null
 
 func updateMainCharacter():
-	if(currentCharacter == null):
-		nameLabel.text = ""
-		nameLabel.self_modulate = Color.white
-		return
-	
 	var character:Character = GlobalRegistry.getCharacter(currentCharacter)
-	if(character == null):
-		nameLabel.text = "???"
-		nameLabel.self_modulate = Color.white
+	# I make in GlobalRegestry what empty string don't emmits erorr loging in this func
+	# Just don't forget to add checking empty characters names in GloabalRegestry
+	
+	if (!is_instance_valid(character)):
+		if currentCharacter == "":
+			nameLabel.text = ""
+			nameLabel.self_modulate = Color.white
+		else:
+			nameLabel.text = "???"
+			nameLabel.self_modulate = Color.white
+		
 		return
+		
 	nameLabel.text = character.getName()
 	nameLabel.self_modulate = character.getChatColor()
 	character.updateEffectPanel(characterStatusGrid)
 
 func _on_NextButton_pressed():
 	var keys = characters.keys()
 	
-	if(currentCharacter == null && keys > 0):
-		currentCharacter = keys[0]
-	
-	if(currentCharacter == null):
+	if (currentCharacter == ""):
 		updateMainCharacter()
-		return
+		if (keys > 0): currentCharacter = keys[0]
+		else: return
```

Comment:
This part of the code is a bit messy but you made it even more messy with that, new lines aren't bad

Reply:
Two "If"-s gets more processing time then one. Logically it's same, but a bit faster.